app/assets/javascripts/discourse/app/components/modal/flag.hbs

```diff
@@ -8,6 +8,11 @@
 >
   <:body>
     <p>{{i18n "flagging.review_process_description"}}</p>
+    <PluginOutlet
+      @name="flag-modal-review-process-description"
```

Comment:
```suggestion
      @name="after-flag-modal-review-process-description"
```

Any objection to prefixing this with "after-" so it is clear that it doesn't override the description but adds elements after it?